packages/manager/apps/telecom/src/app/telecom/pack/move/offers/move-offers.controller.js

```diff
@@ -30,12 +30,28 @@ export default class PackMoveOffersCtrl {
     this.PROMO_DISPLAY = PROMO_DISPLAY;
 
     this.$q
-      .all([this.getOptions(), this.getCopperOffers()])
+      .all([this.getOptions()])
+      .then(() => {
+        if (
+          (this.currentOffer.isFTTH && !this.eligibilityReferenceFiber) ||
+          !this.currentOffer.isFTTH
+        ) {
+          // Retrieve copper offers for xDSL customer or FTTH customer not eligible to fiber
+          return this.getCopperOffers();
+        }
+        return null;
+      })
       .then(() => {
         return this.getFiberOffers();
       })
       .then(() => {
-        if (this.eligibilityReferenceFiber) {
+        if (this.currentOffer.isFTTH && this.eligibilityReferenceFiber) {
+          // For FTTH customer eligible to fiber, display fiber offers ONLY
+          this.offers = [...this.listOffersFiber];
```

Comment:
No need to spread only one array
```suggestion
          this.offers = this.listOffersFiber;
```